skrf/network.py

```diff
@@ -336,6 +337,9 @@ def __init__(self, file=None, name=None, comments=None, f_unit=None, **kwargs):
             its a str
         comments : str
             Comments associated with the Network
+        s_definition : str
```

Comment:
`s_def` would be shorter. but i leave it up to you.

Reply:
OK, I agree